spec/requests/profile_pages_spec.rb

```diff
@@ -28,7 +28,8 @@
         expect(page).to have_link('Edit Profile')
         expect(page).to have_link('View Orders')
 
-
+        click_link('View Orders')
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.